core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -891,8 +891,8 @@ public NativeImageInvokerInfo build() {
 
                 for (ExcludeConfigBuildItem excludeConfig : excludeConfigs) {
                     nativeImageArgs.add("--exclude-config");
-                    nativeImageArgs.add(excludeConfig.getJarFile());
-                    nativeImageArgs.add(excludeConfig.getResourceName());
+                    nativeImageArgs.add("\"" + excludeConfig.getJarFile() + "\"");
```

Comment:
@zakkak I'm curious to see what CI thinks of it. I suspect it won't work on all kinds of Windows terminals as I had tried something similar in the past -  I'm not sure if I had tried *exactly* the same thing.

Reply:
This turns out to be more complex that I thought. The `\"` works fine on windows but not on linux. 
As a result, for a quick-fix I will switch to my previous approach that was not relying on the quotes at the cost of using more expressions than one and avoiding the use of `.*`.

Reply:
Ok, I did remember trying with double quotes, but didn't remember why it doesn't work :)

I assume we're back to using two separate pattern exclusions?

Reply:
> using two separate pattern exclusions?

Yes, I will try to see why the quotes behave differently in windows than linux and follow up if I find anything actionable